fault/subprocess_run.py

```diff
@@ -130,18 +127,23 @@ def subprocess_run(args, cwd=None, env=None, disp_type='info', err_str=None,
     with Popen(args, cwd=cwd, env=env, stdout=PIPE, stderr=PIPE, bufsize=1,
                universal_newlines=True, shell=shell) as p:
 
-        # process STDOUT, then STDERR
+        # print out STDOUT, then STDERR
         # threads could be used here but pytest does not detect exceptions
-        # in child threads, so for now the outputs are processed sequentially
-        stdout = process_output(fd=p.stdout, err_str=err_str,
-                                disp_type=disp_type, name='STDOUT')
-        stderr = process_output(fd=p.stderr, err_str=err_str,
-                                disp_type=disp_type, name='STDERR')
+        # in child threads, so for now the outputs are printed sequentially
+        stdout = process_output(fd=p.stdout, disp_type=disp_type,
+                                name='STDOUT')
+        stderr = process_output(fd=p.stderr, disp_type=disp_type,
+                                name='STDERR')
 
         # get return code and check result if desired
         returncode = p.wait()
         if chk_ret_code:
-            assert not returncode, f'Got non-zero return code: {returncode}'
+            assert not returncode, f'Got non-zero return code: {returncode}.'
```

Comment:
This needs to print out the stdout and stderr. Verilator is returning with a non-zero error code when expects is wrong. My change printed out stderr and stdout. I will add my change back